### PR TITLE
[TestMigration] Migrating test_snap_invlid_cases

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
[Test](https://github.com/gluster/glusto-tests/blob/master/tests/functional/snapshot/test_snap_invalid_cases.py)
    
Updates: #292
    
Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

**Comment by srijan-sivakumar** [comment]
```console
(redant-demo) [root@rhsdev-virt05 redant]# python3 core/redant_main.py -c ../rconfig.yml -t tests/functional/snapshot/test_snap_invalid_cases.py -ll D
    ____  __________  ___    _   ________
   / __ \/ ____/ __ \/   |  / | / /_  __/
  / /_/ / __/ / / / / /| | /  |/ / / /   
 / _, _/ /___/ /_/ / ___ |/ /|  / / /    
/_/ |_/_____/_____/_/  |_/_/ |_/ /_/     
                                         

✔ Param Handling Success.
✔ Test List built
✔ Log dir creation successful.
✔ Environment setup successful.
✔ Running test case : test_snap_invalid_cases-rep
✔ test_snap_invalid_cases-rep Succeeded
✔ Running test case : test_snap_invalid_cases-dist
✔ test_snap_invalid_cases-dist Succeeded
✔ Running test case : test_snap_invalid_cases-disp
✔ test_snap_invalid_cases-disp Succeeded
✔ Running test case : test_snap_invalid_cases-dist-rep
✔ test_snap_invalid_cases-dist-rep Succeeded
✔ Running test case : test_snap_invalid_cases-dist-disp
✔ test_snap_invalid_cases-dist-disp Succeeded
Results for snapshot
Test nature : disruptive
TestName : test_snap_invalid_cases
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:49       |      NA     |
|     dist    |     PASS    |        00:00:44       |      NA     |
|     disp    |     PASS    |        00:00:47       |      NA     |
|   dist-rep  |     PASS    |        00:01:01       |      NA     |
|  dist-disp  |     PASS    |        00:01:03       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 04 minutes 37 seconds
✔ Tearing down successful.
```

File: common/ops/gluster_ops/snapshot_ops.py [hunk 1]
```diff
@@ -362,19 +362,25 @@ def get_snap_status(self, node: str, excep: bool = True) -> dict:
             snap_status_dict[snap['name']] = temp_dict
         return snap_status_dict
 
-    def get_snap_status_by_snapname(self, snapname: str, node: str) -> dict:
+    def get_snap_status_by_snapname(self, snapname: str, node: str,
+                                    excep=True) -> dict:
```
Comment (aujjwal-redhat) [inline common/ops/gluster_ops/snapshot_ops.py:366]: nit: excep: bool

File: tests/functional/snapshot/test_snap_invalid_cases.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+"""
+  Copyright (C) 2017-2018  Red Hat, Inc. <http://www.redhat.com>
+
+  This program is free software; you can redistribute it and/or modify
```
Comment (aujjwal-redhat) [inline tests/functional/snapshot/test_snap_invalid_cases.py:4]: nit : indent

File: tests/functional/snapshot/test_snap_invalid_cases.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Description:
+
+Test Cases in this module tests the
+snapshot Status and Info for Invalid cases.
+
+"""
+
+# disruptive;rep,dist,disp,dist-rep,dist-disp
+
```
Comment (aujjwal-redhat) [inline tests/functional/snapshot/test_snap_invalid_cases.py:26]: I think we should add todo for nfs and cifs. If not here, at least in the sheet in TC Todos.

File: common/ops/gluster_ops/snapshot_ops.py (hunk 1 above)
Comment (srijan-sivakumar) [reply common/ops/gluster_ops/snapshot_ops.py:366]: Ack.

File: tests/functional/snapshot/test_snap_invalid_cases.py (hunk 3 above)
Comment (srijan-sivakumar) [reply tests/functional/snapshot/test_snap_invalid_cases.py:26]: Ack.

File: tests/functional/snapshot/test_snap_invalid_cases.py (hunk 2 above)
Comment (srijan-sivakumar) [reply tests/functional/snapshot/test_snap_invalid_cases.py:4]: I guess we can do away without this change...

Comment (aujjwal-redhat) [reply tests/functional/snapshot/test_snap_invalid_cases.py:4]: Ack

**Review by aujjwal-redhat — APPROVED** [review]

**Pull request merged**